app/views/friendships/_next_page.html.erb

```diff
@@ -1,7 +1,7 @@
-<%= render partial: 'friendships/friends', locals: { friends: friends } %>
+<%= render partial: 'friendships/friendships', locals: { friendships: friendships } %>
```

Comment:
this seems wrong

Reply:
Yea I must have changed this while trying to get turbo_stream working for endless scrolling of friendships on profile pages. I'm pretty sure this file can get 🪓  actually, I don't think it's being used anywhere...